java/integTest/src/test/java/glide/TransactionTestUtilities.java

```diff
@@ -102,6 +102,7 @@ public static BaseTransaction<?> transactionTest(BaseTransaction<?> baseTransact
         baseTransaction.scard(key7);
         baseTransaction.sismember(key7, "baz");
         baseTransaction.smembers(key7);
+        baseTransaction.sinter(new String[] {key7});
```

Comment:
thats cheating 😉

Reply:
A lot of our transaction tests are very crude.  Although, getting the intersection of a single items is a little too no-op.  Maybe we can update this test to check the intersection of itself?  
```
        baseTransaction.sinter(new String[] {key7, key7});
```

Reply:
No I agree with you, sending it twice is great :)